be/src/olap/tablet.h

```diff
@@ -438,7 +438,8 @@ class Tablet final : public BaseTablet {
                                  const TabletColumn& tablet_column,
                                  vectorized::MutableColumnPtr& dst);
 
-    Status fetch_value_through_row_column(RowsetSharedPtr input_rowset, uint32_t segid,
+    Status fetch_value_through_row_column(RowsetSharedPtr input_rowset,
```

Comment:
Add comments on this methods to explain some critical parameters, emphasize that `cids` should be consistent with `tablet_schema`.